050-AIS-APIManagementwithFunctions/Student/Challenge-02.md

```diff
@@ -0,0 +1,39 @@
+# Challenge 02 - Deploy your Integration Environment
+
+
+[<Previous Challenge](./Challenge-01.md) - **[Home](../README.md)** - [Next Challenge>](./Challenge-03.md)
+
+## Pre-requisites
+
+- You should have completed Challenge 01
+
+## Introduction
+Now that you've created your Bicep templates, you would like to create a CI/CD pipeline to deploy these templates to your Azure environment. 
+
+## Description
+- Create a repository and upload your Bicep templates to the main branch.
+- Create a CI/CD pipeline using either GitHub Actions or Azure DevOps.
+    - If using GitHub actions, a sample workflow (deploy.yml) can be found at Student/Resources/Challenge-02 in the Files tab of WhatTheHack - AIS Teams channel.
+    - If using Azure Pipelines, a sample YAML pipeline (azure-pipelines.yaml) can be found at Student/Resources/Challenge-02 in the Files tab of WhatTheHack - AIS Teams channel.
```

Comment:
"can be found in the `/Challenge-02` folder of the `Resources.zip` file provided by your coach."

Reply:
Done